cogniopenapp/lib/ui/videoScreen.dart

```diff
@@ -7,6 +7,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter/scheduler.dart';
 import 'package:video_player/video_player.dart';
 import 'package:path_provider/path_provider.dart';
+import "../src/address.dart";
```

Comment:
We should have absolute imports instead of relative imports, this would protect us from a refactor breaking it.

Reply:
Thanks for that suggestion.  I've made it.  I also added another comment to address.dart.